charts/newrelic-infrastructure/Chart.yaml

```diff
@@ -1,17 +1,19 @@
 apiVersion: v1
 description: A Helm chart to deploy the New Relic Infrastructure Agent as a DaemonSet
 name: newrelic-infrastructure
-version: 2.7.3
-appVersion: 2.8.3
+version: 2.8.0
+appVersion: 2.9.0
```

Comment:
I know this PR is open automatically by some webhook but I cannot find this image: https://github.com/newrelic/infrastructure-bundle/releases

I am looking the wrong repository?

Reply:
this is the release https://hub.docker.com/r/newrelic/infrastructure-k8s/tags

The infrastructure bundle is used as base image of the one including nri-kubernetes